styles.css

```diff
@@ -132,11 +126,12 @@
 .unique {
   display: flex;
   justify-content: center;
+  color: white;
```

Comment:
Please this color is too big, resize it to be smaller, that's because in fact is not a header.

Reply:
- [ ] You should use clear suggestive sentences. For example, what does the phrase `this color is too big` mean? Instead you could have mentioned what color code you want it to be changed to. In addition to that the reason for the change `in fact is not a header` is not clear either. What would you change if you received this feedback yourself? So, you should use convey clearly in a polite manner in a way that helps the receiver know what the issue is and what needs to be done.
For example, you could rephrase your feedback as `You should use the color of #e5e5e5 for this section as stated in the design/requirement.` or `You should change the color of this section to make it more accessible`, etc